src/layer/__tests__/auth.ts

```diff
@@ -1,10 +1,10 @@
-import 'jest-setup';
+import '../../../jest-setup';
 import axios from 'axios';
 import MockAdapter from 'axios-mock-adapter';
 
 import { constructFixtureGetMap } from './fixtures.auth';
-import { ApiType, setAuthToken, requestAuthToken } from 'src';
-import { invalidateCaches } from 'src/utils/Cache';
+import { ApiType, setAuthToken, requestAuthToken } from '../../index';
+import { invalidateCaches } from '../../utils/Cache';
```

Comment:
Since `invalidateCaches` is exported in `src/index.ts`, could it be in the import above?

Reply:
Changed

Reply:
Resolving